homeassistant/components/switcher_kis/strings.json

```diff
@@ -1,6 +1,12 @@
 {
   "config": {
     "step": {
+      "user": {
+        "description": "Enter your token here\nFor security reasons, some devices require a token to operate,\nthe token can be acquired from this link: https://switcher.co.il/GetKey/\nExample of Token: zvVvd7JxtN7CgvkD1Psujw==\nNote: minimum firmware version required for the token is 2.31",
```

Comment:
Thanks for updating! 👍

The last part of the description is still not quite clear to me.

By "minimum firmware version required for the token is xxx", do you mean that above this version you will need a token, or that if you have a token you need to upgrade firmware first?

Maybe rephrase to be from the point of view of the user, e.g. "If your firmware version is x.xx or above, you must provide a token."

Or

"If your firmware version is x.xx or below, a token is not required."


Even better would be a way to change the text/optional to something specific if you can automatically read the firmware version before this prompt.

Reply:
Updated description :)